pysat/tests/test_meta.py

```diff
@@ -105,63 +108,6 @@ def eval_meta_settings(self, isfloat=True):
                     lkey.__repr__(), self.meta[self.dval, lkey].__repr__(),
                     self.default_val[lkey].__repr__())
 
-        assert 'children' not in self.meta.data.columns
-        assert self.dval not in self.meta.keys_nD()
-        return
-
-    def eval_ho_meta_settings(self, meta_dict):
```

Comment:
Another aspect of era ending though this one drops a whole bunch of weight from meta. Should be good!